apps/emqx_mt/src/emqx_mt_state.erl

```diff
@@ -26,21 +26,34 @@
     clear_self_node/0
 ]).
 
+-export([
+    set_limiter_config/3,
+    get_limiter_config/2,
+    delete_limiter_config/2
+]).
+
 -include("emqx_mt.hrl").
 -include_lib("stdlib/include/ms_transform.hrl").
 -include_lib("snabbkaffe/include/snabbkaffe.hrl").
 
+%%------------------------------------------------------------------------------
+%% Type declarations
+%%------------------------------------------------------------------------------
+
 -define(EMQX_MT_SHARD, emqx_mt_shard).
 
 %% mria tables
 -define(RECORD_TAB, emqx_mt_record).
 -define(COUNTER_TAB, emqx_mt_counter).
 -define(NS_TAB, emqx_mt_ns).
+-define(LIMITER_TAB, emqx_mt_limiter).
```

Comment:
there can be other configs in the future, maybe give it a more generic name ?
e.g. `emqx_mt_config`, and limiter being one of the configs.

Reply:
:checkered_flag: